api/src/services/replication-limit-log.js

```diff
@@ -0,0 +1,93 @@
+const moment = require('moment');
+const db = require('../db');
+const logger = require('../logger');
+
+const LOG_TYPES = {
+  replicationCount: 'replication-count-'
+};
+const LOG_COUNT_DIFF = 100;
+const LOG_MONTH_DIFF = 1;
```

Comment:
It's not immediately clear what these constants represent. I can't think of a better name for them but maybe add a comment?

Reply:
Yeah, I also struggled with naming these 2. Fixed. Comment added.